backend/api/visualization.proto

```diff
@@ -69,6 +69,7 @@ service VisualizationService {
 // for determining output path.
 message CreateVisualizationRequest {
   Visualization visualization = 1;
+  string namespace = 2;
```

Comment:
@chensun I tried running https://github.com/kubeflow/pipelines/blob/master/backend/api/generate_api.sh, but it does not add namespace argument to `visualization.swagger.json`, thus frontend client does not get the new field. Can you help me troubleshoot what went wrong here?

e.g. in the
```
rpc CreateVisualization(CreateVisualizationRequest) returns (Visualization) {
  option (google.api.http) = {
    ...
    body: "visualization"
  };
}
```
What does this `body: visualization` mean? Does it mean we only include the visualization field as request body?

Reply:
Interesting, when I created this PR, I already ran `generate_api.sh` as you can see some files were generated. However, `visualization.swagger.json` was somehow left out. 

I also noticed that Jing mentioned in #3511 the need to manually update swagger.json file, but this wasn't the experience I had last time I made API changes (see #3198).

So I think we might have some regression or bug in swagger, I'll investigate and get back to you.

Reply:
I was wrong, there's no bug but a user error. And you're right that int the rpc definition `    body: "visualization"` means only include visualization field. I'll make a fix.

Reply:
Here's the doc related to this topic: https://cloud.google.com/endpoints/docs/grpc/transcoding